app/views/idv/review/new.html.erb

```diff
@@ -26,20 +26,16 @@
       current_user, url: idv_review_path,
                     html: { autocomplete: 'off', method: :put }
     ) do |f| %>
- 
-  <div class="margin-bottom-4">
```

Comment:
Simplified `.margin-bottom-4 + .margin-top-neg-2` to remove the first and set `.margin-top-2` as class of second.